### DISCUSSION TITLE
Unable to reflash Teensy 3.6 after uploading edrumulus

### DISCUSSION THREAD

**Discussion by zeh235** [corrados/edrumulus#90] [Q&A]
Hi,
i have uploaded edrumulus to a teensy 3.6 and set the USB Type to MIDI. Now i cannot reflash the device, it gets recognized as USB midi device (called edrumulus), so somehow it seems to be working. Arduino IDE tells me to press the button to get into programming mode manually, but pressing the button only reboots the teensy and it wont enter the programming mode. i also tried to do a 15 seconds long press on the button but it doesnt do anything, as soon as i release the button, the device just reboots and i see it being recognized again in dmesg. Any suggestions? maybe its also a hardware related issue since i removed the audioboard which was soldered to the teensy. AFAIK all solder is gone and there should not be any short. any suggestions?? do i have to reset the teensy using the reset pin?
thanks in advance
zeh

**Comment by zeh235** [comment]
well, it's a hardware related issue, when i start bending the board it gets recognized, at least it did for a moment, i think now it's gone, will reflow it and see what happens

**Comment by corrados** [comment]
Teensy 3.6 is unfortunately not supperted by Edrumulus. Just Teensy 4.0 and Teensy 4.1. The processor of the Teensy 3.6 board is too slow for Edrumulus.
